### PR TITLE
Adjust clang format to align trailing comments

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Thoughts?

### CODE REVIEW

File: NorthstarDedicatedTest/convar.h [hunk 1]
```diff
@@ -136,14 +136,14 @@ class ConVar
...
-}; // Size: 0x0080
+	const char* m_pszDefaultValue {};	// 0x0040
+	CVValue_t m_Value {};				// 0x0048
+	bool m_bHasMin {};					// 0x005C
+	float m_fMinVal {};					// 0x0060
+	bool m_bHasMax {};					// 0x0064
+	float m_fMaxVal {};					// 0x0068
+	void* m_pMalloc {};					// 0x0070
+	char m_pPad80[10] {};				// 0x0080
+};										// Size: 0x0080
```
Comment (emma-miler) [inline NorthstarDedicatedTest/convar.h:147]: No

File: NorthstarDedicatedTest/gameutils.h [hunk 2]
```diff
@@ -164,10 +164,10 @@ enum EngineQuitState
...
 {
 	DLL_INACTIVE = 0, // no dll
-	DLL_ACTIVE, // engine is focused
-	DLL_CLOSE, // closing down dll
-	DLL_RESTART, // engine is shutting down but will restart right away
-	DLL_PAUSED, // engine is paused, can become active from this state
+	DLL_ACTIVE,		  // engine is focused
+	DLL_CLOSE,		  // closing down dll
+	DLL_RESTART,	  // engine is shutting down but will restart right away
+	DLL_PAUSED,		  // engine is paused, can become active from this state
```
Comment (emma-miler) [inline NorthstarDedicatedTest/gameutils.h:170]: This does not work

File: NorthstarDedicatedTest/maxplayers.cpp [hunk 3]
```diff
@@ -169,27 +169,27 @@ auto RandomIntZeroMax = (__int64(__fastcall*)())0;
...
+	int v15;					  // eax
+	__int64 v16;				  // r8
+	int v17;					  // edx
+	char v18;					  // r15
+	char v19;					  // bp
+	int v20;					  // esi
+	__int64* v21;				  // rdi
+	__int64 v22;				  // rcx
+	bool v23;					  // al
+	__int64 v24;				  // rax
```
Comment (emma-miler) [inline NorthstarDedicatedTest/maxplayers.cpp:192]: Valve pls fix

**Comment by GeckoEidechse** [comment]
Closing myself as stale. Will maybe investigate again in the future ^^

**Pull request closed without merging**